Tests/TestingTests/ABIEntryPointTests.swift

```diff
@@ -54,7 +54,7 @@ struct ABIEntryPointTests {
   ) async throws -> CInt {
 #if !SWT_NO_DYNAMIC_LINKING
     // Get the ABI entry point by dynamically looking it up at runtime.
-    let copyABIEntryPoint_v0 = try withTestingLibraryImageAddress { testingLibrary in
+    let copyABIEntryPoint_v0 = try _withTestingLibraryImageAddress { testingLibrary in
```

Comment:
This code is in a test file. We don't otherwise bother with underscores in the test target.